compose/services/docker-compose.fuelrod-new.yml

```
@@ -43,6 +44,19 @@
+  legacy:
+    image: tsobu/fuelrod-legacy-portal:${LEGACY_PORTAL}
+    hostname: legacy
+    container_name: legacy
+    restart: unless-stopped
+    depends_on:
+      - fuelrod
+    ports:
+      - 3005:80
     networks:
       - web
       - internal
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)